android/src/fabric/java/com/klarna/vectordrawable/VectorDrawableManager.java

```diff
@@ -0,0 +1,61 @@
+package com.klarna.vectordrawable;
+
+import android.widget.ImageView;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.module.annotations.ReactModule;
+import com.facebook.react.uimanager.SimpleViewManager;
+import com.facebook.react.uimanager.ThemedReactContext;
+import com.facebook.react.uimanager.ViewProps;
+import com.facebook.react.uimanager.ViewManagerDelegate;
+import com.facebook.react.uimanager.annotations.ReactProp;
+import com.facebook.react.viewmanagers.RNVectorDrawableManagerDelegate;
+import com.facebook.react.viewmanagers.RNVectorDrawableManagerInterface;
+
+@ReactModule(name = VectorDrawableManagerImpl.NAME)
+public class VectorDrawableManager extends SimpleViewManager<ImageView>
+		implements RNVectorDrawableManagerInterface<ImageView> {
```

Comment:
Mixing of spaces and tab indentation in this file, please make sure all java files are using 4 spaces